proposals/privacy.md

```diff
@@ -103,23 +102,28 @@ a particular person.  We will call these "privacy sensitive contexts".
 
 ### TDs MUST not contain PII.
 For example, a TD must not contain the email of a person with which it is associated or embed their name, address, or other PII 
-in a description or title field.
+in a description or title field, unless specifically required by the use cases. 
+ML: It must be under the descretion of the owner of the thing to chose to whom to expose which PII.
+There are cases where a TD without PII is useless.
 
 ### TDs MUST be transmitted only to Authenticated and Authorized consumers.
 Services providing TDs must require authentication (for example, using OAuth or PKI) of the requester to ensure they
 are who they say they are.  In addition, the system needs to ensure that the requester is authorized to receive TDs.
 
 ### Consumers MUST accept TDs only from Authenticated producers
-In other words, communication of TDs between producers and consumers must be over mutually authenticated channels.
+In other words, communication of TDs between producers and consumers must be over mutually authenticated channels,
+or the authenticity of a producer is validated by some other means (e.g. cryptographic signatures).
 
 ### TDs provided to a Consumer MUST be filtered to remove any optional information not required by that Consumer.
 For example, if a Consumer does not need human-readable titles or descriptions in a TD, these should be removed.
+ML: How does a producer know about the capabilities / requirements of a consumer? We don't have a mechanism to convey this information. In many scenarios a TD producer does not know about the potential consumers. This comment applies also o the points below.
 If a Consumer does not need to maintain state related to the consumed device, then an ID is not necessary and should be omitted.
 If a Consumer does not do semantic processing, then all semantic annotations can be omitted.
+ML: It still is useful for documentation purposes and should be preserved.
 If a Consumer does not do link processing then links should be omitted.
 Metadata (such as TD creation time) should be omitted if the Consumer does not need it.
 If the Consumer already "knows" the Data Schema or does not process the data (for example, if it only caches it) then data schemas can
-be omitted; likewise for URI templates.
+be omitted; likewise for URI templates. ML: How is this "knowledge" conveyed to the consumer?
```

Comment:
Query parameters in a directory access.   Which we need to define in a new spec.   This is basically the requirements for a directory/discovery service.   What we will do in the "Discovery" spec is specify a normative mechanism that meets these requirements, but other mechanisms that also meet the requirements would also be permitted.